src/content/api/cli.md

```diff
@@ -31,20 +32,101 @@ For proper usage and easy distribution of this configuration, webpack can be con
+| `--watch`       | boolean         | Watch for file changes                                                                                         |
+| `--hot`         | boolean         | Enables Hot Module Replacement                                                                                 |
+| `--no-hot`      | boolean         | Disables Hot Module Replacement                                                                                |
+| `--devtool`     | string          | Controls if and how source maps are generated.                                                                 |
+| `--prefetch`    | string          | Prefetch this request                                                                                          |
+| `--json`        | boolean, string | Prints result as JSON or store it in a file                                                                    |
+| `--mode`        | string          | Defines the mode to pass to webpack                                                                            |
+| `--version`     | boolean         | Get current version                                                                                            |
+| `--stats`       | boolean, string | It instructs webpack on how to treat the stats                                                                 |
+| `--no-stats`    | boolean         | Disables stats output                                                                                          |
+| `--analyze`     | boolean         | It invokes `webpack-bundle-analyzer` plugin to get bundle information                                          |
+
+### Aliases
+
+We've set certain aliases for commonly used flags to make it convenient to work with webpack.
+
+| Shortcut | Replaces        |
+| -------- | --------------- |
+| -c       | `--config`      |
+| -m       | `--merge`       |
+| -o       | `--output-path` |
+| -t       | `--target`      |
+| -w       | `--watch`       |
+| -h       | `--hot`         |
+| -d       | `--devtool`     |
+| -j       | `--json`        |
+| -v       | `--version`     |
+
+### Negated Flags
+
+| Flag       | Description                                                   |
+| ---------- | ------------------------------------------------------------- |
+| --no-color | Disabled any color on the console                             |
+| --no-hot   | Disabled hot reloading if you have it enabled via your config |
+| --no-stats | Disables any compilation stats emitted by webpack             |
+
+### Core Flags
+
+Starting CLI v4 and webpack v5, CLI imports the entire configuration schema from webpack core to alow tuning almost every property from the command line.
+
+Here's the list of all the core flags supported by webpack v5 with CLI v4 - [link](https://github.com/webpack/webpack-cli/tree/next/packages/webpack-cli#webpack-5)
+
```

Comment:
Can we highlight it and dive small example?

Reply:
Done